src/views/HomeView.tsx

```diff
@@ -50,24 +52,24 @@ export const HomeView = () => {
                 </Text>
                 <AutoComplete
                     searching={searching}
-                    placeholder="Search data here"
+                    placeholder="Nom de la base de données"
                     width="100%"
                     size="large"
                     options={options}
                     onSearch={searchHandler}
                 />
 
                 <Text h2 style={{ margin: '30px 0', textAlign: 'center' }}>
-                    Categories
+                    Catégories
                 </Text>
 
                 <div className="categories">
-                    {datasets.map(({ category_name, data }, i) => (
-                        <div className="category" key={i}>
-                            <Text h4>{category_name}</Text>
+                    {datasets.map(({ category, data }, index) => (
+                        <div className="category" key={index}>
+                            <Text h4>{category}</Text>
                             {data.map(({ name, path }, i) => (
```

Comment:
This list needs to have a unique key, you are using the parent key which is different from the child list
consider using **i**(is not explicit maybe it can be **dataIndex**) not **index**

Reply:
Sure, not problem for changing it `dataIndex`